src/main/java/de/yggdrasil/core/dal/requests/implementations/PlayerDataWriteRequest.java

```diff
@@ -0,0 +1,30 @@
+package de.yggdrasil.core.dal.requests.implementations;
+
+import de.yggdrasil.core.dal.data.datasources.PlayerDataDataSource;
+import de.yggdrasil.core.dal.data.datasources.models.PlayerData;
+import de.yggdrasil.core.dal.requests.DALWriteRequest;
+
+
+/**
+ * Represents a write request for PlayerData objects to be written to the PlayerDataDataSource (overwrites existing playerdata for the player contained in the data object).
+ */
+public class PlayerDataWriteRequest implements DALWriteRequest<PlayerDataDataSource, PlayerData> {
+
+    private final PlayerData playerData;
+
+    public PlayerDataWriteRequest(PlayerData playerData){
+        this.playerData = playerData;
+    }
+
+    // the return value of this method is not used in the pipeline
+    @Override
+    public String getKey() {
+        return null;
+    }
```

Comment:
Maybe do something like this instead
```suggestion
    // the return value of this method is not used in the pipeline
    @Override
    public String getKey() {
        throw new java.lang.UnsupportedOperationException();
    }
```

or if the key is just the `UUID` then maybe just return it, even if it's not currently used. This might cause problems in the future if you just return `null`